src/main.cpp

```diff
@@ -149,6 +149,13 @@ namespace
 	{
 		try {
 			auto* input = boost::any_cast<modAVUMetadataInp_t*>(*std::next(std::begin(_rule_arguments), 2));
+
+			if (!strcmp(input->arg0, "rmw")) {
```

Comment:
Let's use `== 0` instead of the negation operator (`!`).

The negation operator takes more effort to understand when used with functions like `strcmp` IMO.